src/index.ts

```diff
@@ -88,6 +94,10 @@ export async function resolveSchema (config: Config): Promise<Schema> {
               })
           }
         })
+     case 'schema':
+       const schemaSource = require(resolve(config.file))
```

Comment:
I mean, I disagree but if you want to make the change go for it.

I think it's more confusing to break with convention. Typically a path starting in a slash means an absolute path, and joining the provided path with the cwd would break that use case.